docs/source/details/config_layout.json

```
@@ -0,0 +1,5 @@
+{
+    "adios": "put ADIOS config here",
+    "adios2": "put ADIOS2 config here",
+    "hdf5": "put HDF5 config here"
```

Could be relevant, because nlohmann-json also supports a couple of binary JSON representations that we might want to use:
```suggestion
    "hdf5": "put HDF5 config here",
    "json": "put JSON config here"
```